test/powershell/Modules/Microsoft.PowerShell.Utility/Default-Aliases.Tests.ps1

```diff
@@ -27,7 +27,6 @@ Describe "Verify approved aliases list" -Tags "CI" {
         $FullCLR = !$isCoreCLR
         $CoreWindows = $isCoreCLR -and $IsWindows
         $CoreUnix = $isCoreCLR -and !$IsWindows
-        $FullCLR -or $CoreWindows -or $CoreUnix
         $aliasFullList = @{
```

Comment:
My bad. It's defined in the module `PSWorkflow` which is not loaded by default. Let's keep it in the list.